proc_test.go

```diff
@@ -175,3 +178,19 @@ func TestHelloWorld(t *testing.T) {
 	)
 	proc.Run(t)
 }
+
+func TestFrameCount(t *testing.T) {
```

Comment:
@sbinet I am not really happy with the testing, can you suggest anything better?

Reply:
The problem I have with testing is not actually related with the precision of the counting of frames inside a specific time window. I would actually like to test that FrameCount is thread safe but in order to do this I suppose I must find a away to control the execution of Proc.run(). Please also check my  other comment here https://github.com/go-p5/p5/pull/33#discussion_r556106445

Reply:
ah. ok.

one could imagine creating a `testProc` with a `draw` func that retrieves the frame count value.
concurrently, launch a go routine that also retrives that frame count value.
to ensure both goroutines (the `testProc` one, and the other one) are running concurrently, some amount of coordination would need to be put in place, though (either with channels or with `sync.WaitGroup`)

Reply:
Again the problem is not the actual implementation of the test for ensuring thread safety, but the fact that testing against custom test code does not make sense for me. Please check my additional commend here https://github.com/go-p5/p5/pull/33#discussion_r556477259